newIDE/app/src/UI/KeyboardShortcuts/DeprecatedKeyboardShortcuts.js

```diff
@@ -158,18 +158,26 @@ export default class DeprecatedKeyboardShortcuts {
       this.onSearch();
     }
 
+    if (this._isControlPressed() && evt.which === MINUS_KEY) {
+      this.onZoomOut();
+    }
+    if (this._isControlPressed() && evt.which === EQUAL_KEY) {
+      this.onZoomIn();
+    }
+
     if (isMacLike()) {
-      //Mac specific shortcuts -- zooming done differently on windows and linux
-      if (this._isControlPressed() && evt.which === MINUS_KEY) {
+      // Mac specific shortcuts
+      if (evt.which === NUMPAD_SUBTRACT) {
         this.onZoomOut();
       }
-      if (this._isControlPressed() && evt.which === EQUAL_KEY) {
+      if (evt.which === NUMPAD_ADD) {
         this.onZoomIn();
       }
-      if (evt.which === NUMPAD_SUBSTRACT) {
+    } else {
+      if (this._isControlPressed() && evt.which === NUMPAD_SUBTRACT) {
```

Comment:
I think we could leave the numpad plus and minus without control. What do you think?

Reply:
Yeah, I guess I tried to match it up with the Chrome shortcuts unnecessarily. Numpad plus and minus seem better without control. You're right, I'll change it right away.